robocopkaka/bootcamp-29-project
# Feat create add center component

#### What does this PR do?
Create a component for Adding centers and consume the Express endpoint for adding centers
#### Description of Task to be completed?
* Write tests for the component
* Create the component
* Create an action that adds a center
* Create an action creator that is dispatched when a center is created successfully
* Create an API class that is used to send a request to the endpoint for adding centers
* Create a reducer that changes state to reflect the newly added center
#### How should this be manually tested?
You should be able to fill out the `Add Center` form and see the new center in the store when using redux-devtools or any browser tool that lets you inspect state
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?(if applicable)
N/A
